tests/ui/transmutability/alignment/align-fail.rs

```diff
@@ -0,0 +1,23 @@
+// check-fail
```

Comment:
I think it's supposed to fail, the alignment of the source is smaller. I'm going to update the error reporting to actually include the alignments so that we can see what's what